conman/routes/handlers.py

```diff
@@ -53,6 +75,26 @@ class RouteViewHandler(BaseHandler):
 
     Views will receive `route` as a keyword arg.
     """
+    @classmethod
+    def check(cls, route):
+        """Ensure route has a sensible view attribute."""
+        route_name = route.__name__
+        if not hasattr(route, 'view'):
+            return [checks.Error(
+                '{} must have a `view` attribute.'.format(route_name),
+                hint='This is a requirement of {}.'.format(cls.__name__),
+                obj=route,
+            )]
+
+        if not isinstance(vars(route)['view'], staticmethod):
```

Comment:
I think you're right -- `type(route).view` ought to do the trick! I'll remove all this nasty `staticmethod` nonsense! `\o/`